core/Tracker/GoalManager.php

```diff
@@ -724,7 +724,7 @@ protected function recordStandardGoals(VisitProperties $visitProperties, Request
             }
 
             // If multiple Goal conversions per visit, set a cache buster
-            if ($convertedGoal['allow_multiple'] == 0) {
+            if ($convertedGoal['allow_multiple'] === 0) {
```

Comment:
Not sure if we can assume this one to be an integer
```suggestion
            if ((int) $convertedGoal['allow_multiple'] === 0) {
```